Source/WebCore/platform/graphics/texmap/TextureMapperLayer.h

```diff
@@ -287,8 +287,9 @@ class TextureMapperLayer : public CanMakeWeakPtr<TextureMapperLayer> {
 
 #if ENABLE(DAMAGE_TRACKING)
     bool m_damagePropagation { false };
-    Damage m_damage; // In layer coordinate space.
-    Damage m_inferredDamage; // In global coordinate space.
+    Damage m_receivedDamage; // In layer coordinate space.
```

Comment:
Maybe `m_reportedDamage` rather than _received_

Reply:
I've been thinking a bit about it and I think "reported" is ambiguous. Therefore it would be better to extend it - as you suggested - to `m_graphicsLayerReportedDamage`. However, if we think about `TextureMapperLayer` as a whole (it receives updates from its main-thread counterpart) then the `m_receivedDamage` seems to be just fine.

What do you think?

Reply:
It's not a strong preference from my side, so if you're not convinced with `reported` or with the longer name, let's carry on with _received_